assets/js/main.js

```diff
@@ -58,13 +58,14 @@ document.getElementById('showcases-slideshow')
       // Alter count to show all cards
       communityElements.count = participants.length;
       createParticipantesCards(participants, emojis, communityElements, indexes=pageParticipants);
+      communityElements.container.innerHTML = "";
+      createParticipantesCards(participants, emojis, communityElements);
```

Comment:
Currently this is deleting the paginated cards added on line 60, then adding cards for all participants.

These lines:
```
      createParticipantesCards(participants, emojis, communityElements, indexes=pageParticipants);
      communityElements.container.innerHTML = "";
      createParticipantesCards(participants, emojis, communityElements);
```
Probably need to be:
```
      communityElements.container.innerHTML = "";
      createParticipantesCards(participants, emojis, communityElements, indexes=pageParticipants);
```

Reply:
OK, I've made those changes